mkosi/resources/man/mkosi.1.md

```diff
@@ -1130,6 +1130,21 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
     If set to `auto`, either sbsign or pesign are used if available, with sbsign being preferred if both are
     installed.
 
+`Verity=`, `--verity=`
+:   Whether to enforce or disable verity for disk and extension images.
```

Comment:
I'd drop the disks as long as it's not implemented, since this contradicts the second paragraph
```suggestion
:   Whether to enforce or disable verity for extension images.
```